components/card/readme.md

```diff
@@ -58,3 +58,9 @@ Then your style should have:
 
 Please see this example: [Grid Example](https://vivid.vonage.com?path=/story/alpha-components-card--in-a-grid)
 
+## CSS Custom Properties
+
+| Property                       | Default                                          | Description                                      |
+|--------------------------------|--------------------------------------------------|--------------------------------------------------|
+| `--title-line-trim` | none | Controls the number of lines of the title that will be present before trim & ellipsis |
+| `--subtitle-line-trim` | none | Controls the number of lines of the sub-title that will be present before trim & ellipsis |
```

Comment:
why not line-clamp?

Reply:
I find trim more coherent, not all devs know the clam.

Reply:
but that's the actually css property

Reply:
?